FEWD/Week06/lesson12/homework/rock_paper_scissors_starter/js/main.js

```diff
@@ -0,0 +1,87 @@
+var humanScore = 0;
+var computerScore = 0;
+
+function compChoice() {
+  var computerChoice = Math.random();
+  // console.log(computerChoice);
+  if (computerChoice <= 0.3) {
+    return computerChoice = "Rock";
+    // console.log(computerChoice);
+  } else if (computerChoice > 0.3 && computerChoice <= 0.69) {
+    return computerChoice = "Scissors";
+    // console.log(computerChoice);
+  } else {
+    return computerChoice = "Paper";
+    // console.log(computerChoice);
+  }
+}
+
+function userChoiceRock(e) {
+  e.preventDefault();
+  var userChoice = "Rock";
+  var computerChoice = compChoice();
+  // console.log(computerChoice);
+
+  if (userChoice === computerChoice) {
+    // console.log("this is a draw")
+    aDraw();
+  } else if (computerChoice === "Scissors") {
+    // console.log("User wins");
+    userWins();
+    $("#status").html("Computer chose Scissors. User wins");
+  } else {
+    // console.log("Computer wins");
+    computerWins();
+    $("#status").html("Computer chose Paper. Computer wins");
+  }
+}
+
+function userChoicePaper(e) {
+  e.preventDefault();
+  var userChoice = "Paper";
+  var computerChoice = compChoice();
+
+  if (userChoice === computerChoice) {
```

Comment:
So first, well done for recognising that you need a function for each one of the three buttons. That said, the three functions tied to the three buttons' click events could be smaller "wrapper" functions that just pass the right `userChoice` to a larger function that does all the comparisons.

Not this this is wrong, this works beautifully, but you might consider doing something like:

```
function play(userChoice) {
  var computerChoice = compChoice();
  if (userChoice === computerChoice) {
    aDraw();
  } else if /* etc, rest of logic */
}

function userChoicePaper() {
  play("paper");
}

$("#paper").click(userChoicePaper);
```